iotdb-client/cli/src/main/java/org/apache/iotdb/cli/AbstractCli.java

```diff
@@ -881,4 +883,13 @@ static boolean processCommand(CliContext ctx, String s, IoTDBConnection connecti
     }
     return true;
   }
+
+  private static void updateUsingDatabaseIfNecessary(String database) {
+    if (!Objects.equals(usingDatabase, database)) {
+      usingDatabase = database;
+      if (sqlDialect != null && Model.TABLE.name().equals(sqlDialect.toUpperCase())) {
```

Comment:
what if usingDatabase is null? IOTDB_CLI_PREFIX should be IOTDB instead of "IOTDB:"

Reply:
OK, I've changed this part

Reply:
But still wondering in which case the "usingDatabase" will changed from not-null to null, seems we don't have "unuse database" statement for now